qt/scientific_interfaces/ISISReflectometry/GUI/MainWindow/MainWindowPresenter.h

```diff
@@ -57,6 +57,15 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL MainWindowPresenter
   // IMainWindowPresenter overrides
   bool isAnyBatchProcessing() const override;
   bool isAnyBatchAutoreducing() const override;
+  bool isWarnDiscardChangesChecked() const override;
+  bool isCloseEventPrevented() override;
+  bool isCloseBatchPrevented(int batchIndex) const override;
+  bool isOperationPrevented() const override;
+  bool isOperationPrevented(int tabIndex) const override;
```

Comment:
Can this be `private`? Would a more specific name be better, say, something like `isOverwritingBatchAllowed`?